src/seedu/addressbook/commands/EditCommand.java

```diff
@@ -0,0 +1,74 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.*;
+import seedu.addressbook.data.tag.Tag;
+import seedu.addressbook.data.tag.UniqueTagList;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static seedu.addressbook.ui.TextUi.DISPLAYED_INDEX_OFFSET;
+
+public class EditCommand extends Command{
```

Comment:
coding style violation. [read this](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html)